doc/setup/install.rst

```diff
@@ -200,9 +200,9 @@ If you need this feature, you can install the python module like this:
 Fedora
 ''''''
 
-Odoo 11.0 'rpm' package supports Fedora 26.
+Odoo 12.0 'rpm' package supports Fedora 26.
 As of 2017, CentOS does not have the minimum Python requirements (3.5) for
```

Comment:
still of application?

Reply:
Yep, Fedora 26 is still Ok. I also tested with Fedora 28. Fedora 29 is stil beta.

Reply:
@d-fence, I think @mart-e asked here about the CentOS line.

Reply:
Oops, AFAIK, centos 7 is still to old when it comes to python3